app/react/V2/Components/ErrorHandling/ErrorUtils.ts

```diff
@@ -0,0 +1,30 @@
+const handledErrors: { [k: string]: RequestError } = {
+  400: {
+    name: 'Bad Request',
+    message: 'The request could not be processed.',
+    status: 400,
+  },
+  404: {
+    name: 'Not Found',
+    message: "We can't find the page you're looking for.",
+    status: 404,
+  },
+  500: {
+    name: 'Unexpected error',
+    message: 'Something went wrong',
```

Comment:
I think the old string was 'Unexpected error' and has now changed. Just a reminder to check the translations.